src/LibraryManager.Vsix/Shared/VsHelpers.cs

```diff
@@ -222,6 +226,38 @@ public static bool IsKind(this Project project, params string[] kindGuids)
             return false;
         }
 
+        public static async Task<bool> DeleteFilesFromProjectAsync (Project project, IEnumerable<string> filePaths, Action<string, LogLevel> logAction, CancellationToken cancellationToken)
+        {
+            if (IsCapabilityMatch(project, Constants.DotNetCoreWebCapability))
+            {
+                return false;
```

Comment:
Should we return false here?  Does that mean that we failed to delete the files from the project?

Reply:
I think the behavior here is questionable. 
`AddFilesToProjectAsync` method above does not add this file to the project if its a DotNet Core project. 
However, if the user manually includes the file in the project, (for whatever reason), should this method remove it?

Reply:
But you are right. This should not return false.